resources/sass/components/_filters.scss

```diff
@@ -18,31 +27,46 @@
     }
 
     &__subheader {
-        padding: $indent-m;
+        color: $c-gray2;
+        display: flex;
+        padding: $indent-sm;
+
+        &:hover {
+            cursor: pointer;
+        }
 
         &_has-icon {
             &::after {
+                align-self: center;
                 color: $c-gray2;
                 content: 'L';
                 display: inline-block;
+                flex-grow: 1;
                 font-family: $ff-ris-font-icons;
                 font-size: $fs-s;
-                height: 16px;
-                line-height: $lh-s;
+                line-height: 1;
                 margin: 0;
                 padding: 0;
                 pointer-events: none;
-                position: absolute;
-                right: $indent-m;
                 speak: none;
-                top: $indent-m;
-                width: 16px;
+                text-align: right;
+                width: 100%;
             }
 
             #{$base}__subheader {
                 padding-right: $indent-l;
             }
         }
+
+        &_has-left-icon {
+            &::before {
```

Comment:
better to use separate element for icons, for example <span class="..."></span> and not redefine icon on the whole component with :after or :before